app/lib/submission_builder/state1099_r.rb

```diff
@@ -0,0 +1,53 @@
+module SubmissionBuilder
+  class State1099R < SubmissionBuilder::Document
+    include SubmissionBuilder::FormattingMethods
+
+    def document
+      form1099r = @kwargs[:form1099r]
+      state_abbreviation = form1099r.intake.state_code.upcase
+
+      build_xml_doc("IRS1099R", documentId: "IRS1099R-#{form1099r.id}") do |xml|
+        xml.PayerNameControlTxt form1099r.payer_name_control
+        if form1099r.payer_name && form1099r.payer_name != ''
```

Comment:
It would be a bit cleaner to use `.present?`
```suggestion
        if form1099r.payer_name.present?
```